OWNERS_ALIASES

```diff
@@ -56,3 +56,9 @@ aliases:
     - fabianofranz
     - jhixson74
   azure-reviewers:
+  ovirt-approvers:
+    - rgolangh
+    - Gal-Zaidman
+    - dougsland
+    - gekorob
+  ovirt-reviewers:
```

Comment:
let's keep `rgolangh` as approvers list, everybody else should be in reviewers list.

- make sure there are atleast 2 people in reviewers.
- duplicates in approvers and reviewers is okay.

Reply:
done

Reply:
I think we were expecting `rgolangh` to be in both `ovirt-approvers` and `ovirt-reviewers` ?

Reply:
no pb, done